src/EditorFeatures/CSharpTest/UseIndexOrRangeOperator/UseRangeOperatorTests.cs

```diff
@@ -221,6 +219,94 @@ void Goo(string s, string t)
     {
         var v = t.Substring(s[1..^1][0], t.Length - s[1..^1][0]);
     }
+}", parseOptions: s_parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseRangeOperator)]
+        public async Task TestWithTypeWithActualSliceMethod1()
```

Comment:
💡 Add a test for the case where there is also a `Slice(int)` overload, and the value is greater than 0

Reply:
➡️ I misread the spec. This is not necessary.